src/tiled/scriptfileformatwrappers.cpp

```diff
@@ -0,0 +1,128 @@
+/*
+ * scriptfileformatwrappers.h
+ * Copyright 2019, Thorbjørn Lindeijer <bjorn@lindeijer.nl>
```

Comment:
I try to do the per-file copyright notices based on who edited those files, though I know, this is laboursome and error-prone to keep up-to-date (and could be automated based on some git tooling, I guess).

In any case you can put your name here and also feel free to add your name to the list if you make significant changes to other files.